packages/flame_3d/lib/src/graphics/graphics_device.dart

```diff
@@ -48,6 +48,10 @@ class GraphicsDevice {
 
   Size _previousSize = Size.zero;
 
+  /// Must be set by the rendering pipeline before elements are bound.
+  /// Can be accessed by elements in their bind method.
+  Iterable<Light> lights = [];
```

Comment:
Do we want to make this final? And/or private with a `Iterable` getter?

Reply:
cannot be final because it is modified by the world3d.
I opted to make it not final rather than to clear/addAll for performance reasons, but I can change